site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2374,6 +2374,48 @@ public function getAllGradeablesIds() {
         return $this->course_db->rows();
     }
 
+    public function getLimitedGradersAllAccessGradeablesIds() {
```

Comment:
`getGradeableIdsForFullAccessLimitedGraders()` is probably a more grammatically correct way of putting that, not sure we can really shorten it.